db/seeds/seeder.rb

```diff
@@ -0,0 +1,35 @@
+class Seeder
```

Comment:
@citizen428, I pulled out this class so it can be re required in db/seeds.rb as well as my end to end test seed files, but I'm doing something wrong when I try to require it.

For db/seeds.rb, I tried to require it in these two ways with no success.

- `require "./seeder"`
- `require "db/seeder"`

and in my end to end seed file like so with no success either.

`require Rails.root.join("db/seeder")`

The error is pretty clear.

```bash
cy.task('seedData') failed with the following error:

> There was an error running "bundle exec rake db:seed:e2e[admin_login_setup]". The status was 1 with the following error:
rake aborted!
LoadError: cannot load such file -- /Users/nickytonline/dev/forem/db/seeder
/Users/nickytonline/dev/forem/vendor/bundle/ruby/2.7.0/gems/bootsnap-1.5.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:23:in require' /Users/nickytonline/dev/forem/vendor/bundle/ruby/2.7.0/gems/bootsnap-1.5.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:23:in block in require_with_bootsnap_lfi'
/Users/nickytonline/dev/forem/vendor/bundle/ruby/2.7...
```

I imagine that Ruby classes are public by default? I looked at other classes and it seems to be fine the way I put it in it's own `db/seeder.rb` file.

Reply:
Oops, never mind. It's `db/seeds/seeder` not `db/seeder` 🙃 Need coffee.